### PR TITLE
Array

### PR DESCRIPTION
**Pull request by yuliaababiy**

### CODE REVIEW

File: GitExample.cpp [hunk 1]
```diff
@@ -1,12 +1,54 @@
...
+        return 1;
+    }
+
+
+    int size;
+    inputFile >> size;
+
+    int myArray[5];
+
+    for (int i = 0; i < 0; i++) {
```
Comment (Seevka) [inline GitExample.cpp:27]: Please change i<0 to the i<size because your array wont be initializied

File: GitExample.cpp [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+        return 1;
+    }
+    if (!outputFile.is_open()) {
+        cerr << "Unable to open output file.";
+        return 1;
+    }
+
+
+    int size;
+    int n;
```
Comment (Seevka) [inline GitExample.cpp:23]: why do u need n? please delete it

Comment (yuliaababiy) [reply GitExample.cpp:23]: deleted

File: GitExample.cpp (hunk 1 above)
Comment (yuliaababiy) [reply GitExample.cpp]: changed

**Pull request merged**